mmv1/third_party/terraform/tests/resource_privateca_ca_pool_test.go

```diff
@@ -185,7 +187,9 @@ resource "google_privateca_ca_pool" "default" {
         object_id_path = [1, 7]
       }
       ca_options {
+	include_is_ca = true
```

Comment:
Can you add some tests here that set this value to false, as well as setting is_ca to false, and other permutations of these values that can come up? It would be good to have tests for each of them

Reply:
Done.